modules/local/create_samplesheet.nf

```diff
@@ -1,9 +1,6 @@
 process CREATE_SAMPLESHEET {
     tag "$meta.id"
 
-    executor 'local'
-    memory 100.MB
-
```

Comment:
if fetchngs is doing it than it is well tested and seqera approved basically :D. Just had never seen it and am more versed no in using the validation plugin. Okfor me to leave as it was before